dec265/dec265.cc

```diff
@@ -709,8 +722,8 @@ int main(int argc, char** argv)
       }
       else {
         // read a chunk of input data
-        uint8_t buf[BUFFER_SIZE];
-        int n = fread(buf,1,BUFFER_SIZE,fh);
+        uint8_t buf[buffer_size];
```

Comment:
```c
#ifndef (__GNUC__)
        uint8_t buf[BUFFER_SIZE];
        int n = fread(buf,1,BUFFER_SIZE,fh);
#else 
        uint8_t buf[buffer_size];
        int n = fread(buf,1,buffer_size,fh);
#endif 
```

Reply:
@zvezdochiot Is this the workaround for the CI error on MSVC? If so it should explicitly check for that compiler. But in that case there should also not be a way to specify the buffer size (as it won't be used anyway).

I agree with @farindk that we probably can keep a fixed buffer size. Otherwise the configured buffer size should be used on all platforms.

Reply:
@fancycode say:
>Is this the workaround for the CI error on MSVC?

I don’t know another way. For me, only GCC is important. I’m not even familiar with MSVC.